spec/support/helpers/feature_spec_helpers.rb

```diff
@@ -57,7 +55,13 @@ def have_alert(msg, container: 'body')
       click_button "Update #{subject.model_name.human}"
       expect(page).to have_content("Changed #{subject.model_name.human} Name")
       expect(page).to have_content('Record was successfully updated.')
+    end
+  end
 
+  def select_division(division_name)
+    within('.user-div-info') do
+      find('[data-expands="division-dropdown"]').click
+      find('.select_division_form').select(division_name)
```

Comment:
I like the choice to do this refactor. Will selecting no division (the root division) have a division_name of "All Divisions"?